packages/kbn-unified-data-table/src/components/data_table.tsx

```diff
@@ -23,44 +24,41 @@ import {
   EuiIcon,
   EuiDataGridRefProps,
   EuiDataGridInMemory,
+  EuiDataGridControlColumn,
+  EuiDataGridCustomBodyProps,
+  EuiDataGridCellValueElementProps,
 } from '@elastic/eui';
 import type { DataView } from '@kbn/data-views-plugin/public';
-import type { SortOrder } from '@kbn/saved-search-plugin/public';
 import {
   useDataGridColumnsCellActions,
   type UseDataGridColumnsCellActionsProps,
 } from '@kbn/cell-actions';
-import type { AggregateQuery, Filter, Query } from '@kbn/es-query';
-import { FieldFormatsStart } from '@kbn/field-formats-plugin/public';
-import type { ToastsStart, IUiSettingsClient, HttpStart, CoreStart } from '@kbn/core/public';
-import { DataViewFieldEditorStart } from '@kbn/data-view-field-editor-plugin/public';
+import type { ToastsStart, IUiSettingsClient } from '@kbn/core/public';
 import { Serializable } from '@kbn/utility-types';
```

Comment:
since we're changing this area
```suggestion
import type { Serializable } from '@kbn/utility-types';
```